backends/advanced/src/advanced_omi_backend/workers/transcription_jobs.py

```
@@ -624,13 +663,20 @@ async def create_audio_only_conversation(
+        f"✅ Created batch transcription conversation {session_id[:12]} for fallback"
+    )
     return conversation
 
 
 @async_job(redis=True, beanie=True)
 async def transcription_fallback_check_job(
-    session_id: str, user_id: str, client_id: str, timeout_seconds: int = None, *, redis_client=None
+    session_id: str,
+    user_id: str,
+    client_id: str,
+    timeout_seconds: int = None,
```

Ruff (0.15.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)